src/System.Net.Http/tests/FunctionalTests/HttpClientHandlerTest.Cancellation.cs

```diff
@@ -343,6 +343,36 @@ public async Task MaxConnectionsPerServer_WaitingConnectionsAreCancelable()
             }
         }
 
+        [Fact]
+        public async Task SendAsync_Cancel_CancellationTokenPropagates()
```

Comment:
Is the intent to run this test for HTTP2? I don't see where these tests are running for HTTP2 currently.

Also, what about the other cancellation tests here?

Reply:
That was my intent. Let me try to update that.

The other tests here don't abstract from the 1.1 loopback server to the 2.0 loopback server, at least with the current API.

Reply:
This is fixed in the latest iteration.